source/contributing.md

```diff
@@ -7,42 +7,52 @@ version: 2.1
 
 # How to Contribute
 
-Apache Weex is an open source project which is under active development,  developers can participate and contribute to the Apache Weex community in a variety of ways. Contributing doesn't just mean contributing code, reporting bugs, answering questions, joining in discussions and contributing documentation are also welcome.
-
-## Report Bugs
+Apache Weex is an open source project that is under active development and we're continuously working out to simplify contributing to Weex ecosystem. Developers can participate and contribute to the Apache Weex community by reporting bugs, contributing code, answering questions, joining in discussions and contributing to documentation.
 
+## Bugs
 You can use Weex directly through [Get Started](./guide/index.html) tutorial. If you encounter any unexpected phenomena during your use, feel free to let us know by filing a bug report.
```

Comment:
The title of that link is `Getting Started`. Should we change it here, or change the article title?

Reply:
We should change it here. 'Getting Started' makes more sense. I'll make the change and push again.